ext_conf_template.txt

```diff
@@ -51,3 +54,6 @@ keepFEGroups = 0
 
 # cat=frontend/enable; type=boolean; label=LLL:EXT:ig_ldap_sso_auth/Resources/Private/Language/locallang_db.xlf:settings.enableFESSO
 enableFESSO = 0
+
+# cat=frontend/enable; type=boolean; label=LLL:EXT:ig_ldap_sso_auth/Resources/Private/Language/locallang_db.xlf:settings.FESSOKeepDomainName
+FESSOKeepDomainName = 0
```

Comment:
same here: the option does not stick to the existing conventions in use, it should be something like `keepFESSODomainName` instead (or alike).